src/easy_config/__init__.py

```diff
@@ -165,6 +171,21 @@ def load(
         if _additional_files:
             for f in _additional_files:
                 values.update(cls._load_file(f))
+        if _lookup_config_envvar is not None:
+            envvar = f'{cls.NAME.upper()}_{_lookup_config_envvar.upper()}'
+
+            try:
+                f = os.environ[envvar]
+            except KeyError as e:
+                raise KeyError(f'No value set for {envvar} in environment') from e
+
+            if not f:
+                raise ValueError(f'Empty value set for {envvar} in environment')
+
+            if not os.path.exists(f):
```

Comment:
Don't need this, configparser transparently handles nonexistent files.

I'll add this to the documentation for overriding `_load_file` to say that it should never fail.

Unless you think this should be an error?

Reply:
idk, I leave this up to you to decide. If there's invalid configuration for the configuration (meta-configuration?) should it explode? I happen to like programs that explode when it's my fault rather than saying "that's fine, charlie, just go on"

Reply:
I think the possibility of someone having a default configuration location that doesn't need to exist is the better case to optimize for.

Maybe I'll go back and add some `logger.debug`'s for things like that.

Reply:
Okay, fair enough!